examples/onnx/yolov5-videos/cortex.yaml

```diff
@@ -0,0 +1,13 @@
+# WARNING: you are on the master branch, please refer to the examples on the branch that matches your `cortex version`
+
+- name: youtube-yolov5
```

Comment:
See my comment on `README.md` regarding potentially changing this to `yolov5-youtube`